android/src/main/cpp/graphics/shader/BaseShaderProgramOpenGl.cpp

```diff
@@ -68,18 +70,58 @@ void BaseShaderProgramOpenGl::checkGlProgramLinking(GLuint program) {
     }
 }
 
+const GLuint BaseShaderProgramOpenGl::FRAME_UBO_BINDING_POINT = 0;
+
+/*
+ * Returns the uniform block definition for the frame-wide render values.
+ * Caution! The binding location is fixed to BaseShaderProgramOpenGl::. To use this buffer and to avoid collisions with other UBOs, other shaders
```

Comment:
```suggestion
 * Caution! The binding location is fixed to BaseShaderProgramOpenGl::FRAME_UBO_BINDING_POINT. To use this buffer and to avoid collisions with other UBOs, other shaders
```